README.mediawiki

```diff
@@ -981,6 +981,13 @@ Those proposing changes should consider that ultimately consent may rest with th
 | Standard
 | Draft
 |- style="background-color: #ffffcf"
+| [[bip-0324.mediawiki|324]]
+| Peer Services
+| Version 2 P2P Encrypted Transport Protocol
+| Dhruv Mehta, Tim Ruffing, Jonas Schnelli, Pieter Wuille
+| Standard
+| Draft
+|-
```

Comment:
The `style="background-color: #ffffcf"` line needs to be after bip-324, not before it.

Reply:
That fixed it. Thanks, @ajtowns!